package.json

```diff
@@ -72,9 +71,11 @@
     "web3-typescript-typings": "^0.7.2"
   },
   "devDependencies": {
+    "@types/bignumber.js": "^5.0.0",
```

Comment:
I dont think this is needed anymore, big number now bundles its own types

Reply:
Removed